lib/registrations/registration_checker.rb

```diff
@@ -3,6 +3,8 @@
 module Registrations
   class RegistrationChecker
     COMMENT_CHARACTER_LIMIT = 240
+    DEFAULT_GUEST_LIMIT = 1000
```

Comment:
You can expose this value to the frontend via `app/webpacker/lib/wca-data.js.erb`. That way the frontend form can already perform a `max` check on the number input